src/components/CommentReply/index.tsx

```diff
@@ -199,7 +197,7 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
 
     return (
       <>
-        {this.renderAuthorDate()}
+        {this.renderHeader()}
```

Comment:
It does feel like there's a lot of duplication in these methods for the `Comment` and `CommentReply` components - feels like we could get away with just changing the classname and contents of the final div on most